examples/Permissions/Permissions/AppDelegate.swift

```diff
@@ -16,8 +19,27 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
 
-        // Set the global log level like this
-        LogManager.severity = .Notice
+        // Set up Logging with SwiftyBeaver
+
+        let console = ConsoleDestination()
+        log.addDestination(console)
+
+        LogManager.logger = { message, severity, file, function, line in
+            switch severity {
+            case .Verbose:
+                log.verbose(message, file, function, line: line)
+            case .Notice:
+                log.debug(message, file, function, line: line)
+            case .Info:
+                log.info(message, file, function, line: line)
+            case .Warning:
+                log.warning(message, file, function, line: line)
+            case .Fatal:
+                log.error(message, file, function, line: line)
+            }
+        }
+        
+        LogManager.severity = .Info
```

Comment:
Okay, so here is how to configure the internal logger to use an external logging framework such as SwiftyBeaver.

And so now logging output looks like this....

![custom logging](https://cloud.githubusercontent.com/assets/309420/11874525/1b5d988a-a4d8-11e5-8588-74432c7a591f.png)

The only thing that is perhaps a little bit annoying, is that the various severity levels might not always match up. Operations defines:

```swift
public enum LogSeverity: Int, Comparable {

    /// Chatty
    case Verbose = 0

    /// Public Service Announcements
    case Notice

    /// Info Bulletin
    case Info

    /// Careful, Errors Occurring
    case Warning

    /// Everything Is On Fire
    case Fatal
}
```

And, SwiftyBeaver defines:

```swift
    public enum Level: Int {
        case Verbose = 0
        case Debug = 1
        case Info = 2
        case Warning = 3
        case Error = 4
    }
```

So, you'll have to match things up...